netlify.toml

```diff
@@ -4,3 +4,7 @@
     [plugins.inputs]
 
     skipPatterns = ["#!", "www.google-analytics.com", "www.googletagmanager.com"]
+
+[[redirects]]
+  from = "/docs/overview"
+  to = "/docs/concept-overview"
```

Comment:
i think not necessary if you have `_redirects` in the right place

Reply:
The _redirects file needs to be in the build folder - 
With Docusaurus the build folder is created when the site is built.
So, alternative is to use the .toml file - which we already have and doesn't need to be any place other than the root directory.

Reply:
we can also put it in `/static` :)

Reply:
probably, but the .toml file is already there and provides more comprehensive redirect options -
And it works -